pkg/tfbridge/schema_test.go

```diff
@@ -1241,92 +1241,165 @@ func TestInvalidAsset(t *testing.T) {
 }
 
 func TestOverridingTFSchema(t *testing.T) {
-	ctx := context.Background()
+	t.Parallel()
+
+	const (
+		k                 = "prop"
```

Comment:
I'd love for this to be "testProp" hard coded everywhere instead of abstracted here - I find that this var is just far enough away from most of the tests to be not really clear as to why it's called "k".

Reply:
I'm happy to call it testProp and move it closer to where it is used.